src/scss/components/_rate.scss

```diff
@@ -0,0 +1,73 @@
+$rate-color: $grey-lighter !default;
+$rate-color-active: darken($warning, 5%) !default;
+$rate-space: 5px !default;
+
+.rate {
+  align-items: center;
+  display: flex;
+  &:not(:last-child) {
+    margin-bottom: ($block-spacing / 2);
+  }
+  &.is-spaced .rate-item:not(:last-child) {
+    margin-right: $rate-space;
+  }
+  &.is-disabled .rate-item {
+    cursor: initial;
+    &:hover {
+      transform: none;
+    }
+  }
+  .rate-item {
+    cursor: pointer;
+    display: inline-flex;
+    position: relative;
+    transition: all 0.3s;
+    &:hover {
+      transform: scale(1.1);
+    }
+    &.set-on .icon,
+    &.set-half .is-half {
+      color: $rate-color-active;
+    }
+  }
+  .icon {
+    color: $rate-color;
+    pointer-events: none;
+    width: inherit;
+    // fix MDI when half star in all size
+    line-height: 1;
+  }
+  .is-half > i {
+    // This follow width in rate.vue
+    // Because b-icon not support custom style
+    position: absolute;
+    left: 0;
+    top: 0;
+    width: inherit;
+    overflow: hidden;
+    // fix FA not same height + position from size default (fa-lg)
+    &.fa-lg {
+      top: 1.25px;
```

Comment:
Not sure why customizing top is necessary... how is it possible that the same icon have different height?

Reply:
This is my proposal that removed the need for this fix. You will get the same behavior without having a font specific fix.

Add this scss:
```
.rate .rate-item.set-half .is-half {
    position: absolute;
    left: 0;
    top: 0;
    overflow: hidden;
}
```

Remove `.rate .is-half > i.fa-lg` entirely

Change `.rate .is-half > i` to :
```
.rate .is-half > i {
    position: absolute;
    left: 0;
}
```